tests/unit/amo/components/TestConfirmationDialog.js

```diff
@@ -1,118 +1,85 @@
 import * as React from 'react';
+import userEvent from '@testing-library/user-event';
 
-import Button from 'amo/components/Button';
-import ConfirmationDialog, {
-  ConfirmationDialogBase,
-} from 'amo/components/ConfirmationDialog';
-import {
-  createFakeEvent,
-  fakeI18n,
-  shallowUntilTarget,
-} from 'tests/unit/helpers';
+import ConfirmationDialog from 'amo/components/ConfirmationDialog';
+import { render as defaultRender, screen } from 'tests/unit/helpers';
 
 describe(__filename, () => {
-  function render(otherProps = {}) {
-    const props = {
-      i18n: fakeI18n(),
-      onCancel: sinon.stub(),
-      onConfirm: sinon.stub(),
-      ...otherProps,
-    };
-    return shallowUntilTarget(
-      <ConfirmationDialog {...props} />,
-      ConfirmationDialogBase,
-    );
+  function render(props = {}) {
+    return defaultRender(<ConfirmationDialog {...props} />);
   }
 
   it('renders a dialog', () => {
     const message = 'this action is risky, are you sure?';
-    const root = render({ message });
+    render({ message });
 
-    expect(root.find('.ConfirmationDialog-message')).toHaveLength(1);
-    expect(root.find('.ConfirmationDialog-message')).toHaveText(message);
+    expect(screen.getByText(message)).toHaveClass('ConfirmationDialog-message');
```

Comment:
Did you see my comment at https://github.com/mozilla/addons-frontend/pull/11373#discussion_r843959456? I don't think we should be adding assertions for hardcoded class names. We have a ton of code in which we have hardcoded class names that are not asserted about, and I don't think we are about to add assertions for all of them.